spec/renuo/bin-check/initializer_spec.rb

```diff
@@ -29,22 +29,22 @@
       end
 
       begin
-        bin_check.run
+        expect { bin_check.run }.to output("I have been running\nNow I'm tired\n").to_stdout
```

Comment:
is this an integration test or a unit test? i guess it is a unit test, and therefore i wouldn't test if this is the output, but instead if the printer was called with the correct object.